__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -13,6 +13,9 @@ import { ButtonProps, TextAreaProps } from '@/interfaces/taskDetails.type';
 import { ToastContainer } from 'react-toastify';
 import Details from '@/components/taskDetails/Details';
 
+const urlParams = new URLSearchParams(window.location.search);
+const isDevMode = urlParams.get('dev') === 'true';
+
```

Comment:
These 2 variables are not used anywhere

Why do we need them when we are using next router with query as a parameter for render with router

Reply:
Removed them, thanks for pointing out 🙂